src/main/java/heronarts/lx/dmx/DmxRangeModulator.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2023- Justin K. Belcher, Mark C. Slee, Heron Arts LLC
+ *
+ * This file is part of the LX Studio software library. By using
+ * LX, you agree to the terms of the LX Studio Software License
+ * and Distribution Agreement, available at: http://lx.studio/license
+ *
+ * Please note that the LX license is not open-source. The license
+ * allows for free, non-commercial use.
+ *
+ * HERON ARTS MAKES NO WARRANTY, EXPRESS, IMPLIED, STATUTORY, OR
+ * OTHERWISE, AND SPECIFICALLY DISCLAIMS ANY WARRANTY OF
+ * MERCHANTABILITY, NON-INFRINGEMENT, OR FITNESS FOR A PARTICULAR
+ * PURPOSE, WITH RESPECT TO THE SOFTWARE.
+ *
+ * @author Mark C. Slee <mark@heronarts.com>
+ * @author Justin K. Belcher <justin@jkb.studio>
+ */
+
+package heronarts.lx.dmx;
+
+import heronarts.lx.LXCategory;
+import heronarts.lx.modulator.LXModulator;
+import heronarts.lx.parameter.BooleanParameter;
+import heronarts.lx.parameter.DiscreteParameter;
+import heronarts.lx.parameter.LXParameter;
+
+/**
+ * Modulator receiving a range from [min] to [max] within a DMX channel.
+ */
+@LXModulator.Global("DMX Range")
+@LXModulator.Device("DMX Range")
+@LXCategory(LXCategory.DMX)
+public class DmxRangeModulator extends DmxModulator {
+
+  public final DiscreteParameter min =
+    new DiscreteParameter("Min", 0, 256)
+    .setDescription("Minimum input value for range");
+
+  public final DiscreteParameter max =
+    new DiscreteParameter("Max", 255, 0, 256)
+    .setDescription("Maximum input value for range");
+
+  public final BooleanParameter active =
```

Comment:
My typical convention for boolean modulators is to call this variable `out`

Reply:
Done, thx.